src/components/primaryButton/PrimaryButton.jsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
+import { ButtonStyled } from './PrimaryButton.styled';
+import PropTypes from 'prop-types';
+import { ButtonPlus } from 'components';
 
-const PrimaryButton = () => {
-  return <div>PrimaryButton</div>;
+const PrimaryButton = ({
+  children,
+  onClick,
+  onSubmit,
+  width,
+  height,
+  hasIcon,
+  svgName,
+  ...rest
+}) => {
+  
+  return (
+    <ButtonStyled
+      onClick={onClick}
```

Comment:
possible can be a conflict onClick and onSubmit , we need create additional checking